src/features/Classes/ClassDetailPage/index.jsx

```diff
@@ -106,13 +118,28 @@ const ClassDetailPage = () => {
             </Button>
           </div>
           <InstructorCard />
-          <div className="d-flex justify-content-end my-3 flex-wrap">
+          <div className="d-flex justify-content-end my-3 align-items-center ">
             <Button
               onClick={handleEnrollStudentModal}
               className="text-decoration-none text-primary bg-white"
             >
               Invite student to enroll
             </Button>
+            <Dropdown className="dropdowntpz mx-3">
```

Comment:
Add also this menu in other column in class table, the same as here: https://certprepcourseware-staging.pearson.com/certprep-manager/classes